caffe2/CMakeLists.txt

```diff
@@ -460,6 +460,11 @@ if (NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     ${TORCH_ROOT}/test/cpp/jit/test.cpp
     )
 
+  if (INTERN_BUILD_MOBILE)
+    list(REMOVE_ITEM TORCH_SRCS ${TORCH_SRC_DIR}/csrc/jit/export.cpp)
```

Comment:
REMOVE_ITEM is a bit magical, can we instead just add it only if the opposite condition is true?

Reply:
You mean something like this?  
```
  if (NOT INTERN_BUILD_MOBILE)
    list(APPEND TORCH_SRCS ${TORCH_SRC_DIR}/csrc/jit/export.cpp)
    list(APPEND TORCH_SRCS ${TORCH_ROOT}/test/cpp/jit/test.cpp)
  endif()
```